example/custom-metrics-deployment.yaml

```diff
@@ -1,31 +1,31 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: gardener-custom-metrics
+  labels:
+    app: gardener-custom-metrics
+  name: gardener-custom-metrics-debug
   namespace: garden
 spec:
   replicas: 1
   selector:
     matchLabels:
       app: gardener-custom-metrics
       gardener.cloud/role: gardener-custom-metrics
-      resources.gardener.cloud/managed-by-xxx: gardener
+      resources.gardener.cloud/managed-by: gardener
```

Comment:
Do we need this label if this deployment actually does not belong to a MR when you deploy the `garden-custom-metrics` via the example folder.

Reply:
We don't need it. It was a leftover from initial experimentation. Removed.